extension/webviews/process-editor/src/app.ts

```diff
@@ -32,7 +32,8 @@ class IvyGLSPStarter extends GLSPStarter {
       ivyBreakpointModule,
       ivyStartActionModule,
       ivyInscriptionModule,
-      ivyStartupDiagramModule
+      ivyStartupDiagramModule,
+      navigationModule
```

Comment:
we remove the navigationModule in createIvyDiagramContainer(...), but it is needed here.